### DISCUSSION TITLE
im unable to craete a user in SOC

### DISCUSSION THREAD

**Discussion by sandraimmaculate** [Security-Onion-Solutions/securityonion#13043] [Unsupported Versions]
Hi , 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/0a03ffc2-73e7-4180-b7b1-0b41626bdf26)

in hunt i'm getting like 
message
	{"fields":{"contentLength":0,"method":"POST","status":"200 OK","statusCode":200,"url":"https://so.on.com/sensoroniagents/api/node"},"level":"info","timestamp":"2024-05-21T07:49:56.274233968Z","message":"HTTP request finished"}

**Comment by reyesj2** [comment]
What version are you using?

Does ```so-status``` return all services as 'OK'?

**Reply by sandraimmaculate** [reply]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/a765c8e8-d5a0-4144-9064-5fae66a8e74a)
from cli i'm able to create user but not from web  i'm using the version 2.4

**Comment by dougburks** [comment]
@sandraimmaculate From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Community Support is considered best effort and there are no guarantees and no SLAs. **Please do not mark discussions as urgent.** If you need urgent, private, priority, or enterprise support, please consider purchasing support from [Security Onion Solutions](https://securityonionsolutions.com/support).
